Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -105,6 +105,7 @@ extension Command {
         var args = module.basicArgs
         args += module.optimizationFlags(conf)
         args += ["-L\(prefix)"]
+        args += module.languageArgs
```

Comment:
Ah, I missed that. Yeah we should clearly distinguish between linking arguments and compiling arguments.